vision/spec/detect_crop_hints_spec.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in write, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud/storage"
+
+require_relative "../detect_crop_hints"
+
+describe "Detect Crop Hints" do
+
+  before do
+    @storage    = Google::Cloud::Storage.new
```

Comment:
This is not necessary in the tests.

We do not currently do this in tests.

    $ git grep "Google::Cloud::Storage.new" | grep spec

```ruby
bigquery/spec/bigquery_sample_spec.rb:    @storage    = Google::Cloud::Storage.new
language/spec/language_samples_spec.rb:    @storage     = Google::Cloud::Storage.new
logging/spec/sample_spec.rb:    @storage    = Google::Cloud::Storage.new
storage/spec/buckets_spec.rb:    @storage     = Google::Cloud::Storage.new
storage/spec/files_spec.rb:    @storage         = Google::Cloud::Storage.new
storage/spec/quickstart_spec.rb:    storage     = Google::Cloud::Storage.new
vision/spec/detect_crop_hints_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_document_text_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_faces_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_image_properties_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_labels_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_landmarks_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_logos_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_safe_search_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_text_spec.rb:    @storage    = Google::Cloud::Storage.new
vision/spec/detect_web_spec.rb:    @storage    = Google::Cloud::Storage.new
```

When we remove all project IDs, we will be able to remove them from samples only, as tests use the GOOGLE_CLOUD_PROJECT variable.